languagetool-server/src/main/java/org/languagetool/server/ExtendedUserInfo.java

```diff
@@ -44,13 +46,20 @@ public class ExtendedUserInfo {
   public UUID groupId;
   public String groupRole;
 
+  @Getter
+
+  public boolean opt_in_3rd_party_ai_grammar_checker;
+
+  @Getter
+  public boolean opt_in_3rd_party_ai_paraphraser;
+
 
   // for jackson-databind deserialization
   public ExtendedUserInfo() {}
 
   // for myBatis deserialization
   public ExtendedUserInfo(String addon_token, String api_key, String email, String name, Date premium_from, Date premium_to, Timestamp cancel_date, Long subscription_months, String geo_ip_country,
-   Long managed_accounts, UUID groupId, String groupRole) {
+   Long managed_accounts, UUID groupId, String groupRole, boolean opt_in_3rd_party_ai_grammar_checker, boolean opt_in_3rd_party_ai_paraphraser) {
```

Comment:
Any reason not using camelCase for `opt_in_3rd_party_ai_grammar_checker` and `opt_in_3rd_party_ai_paraphraser` here and at other places?

Reply:
The fields here at named according to the database fields for serialization / deserializiation.